### DISCUSSION TITLE
Welcome to neurodesk Discussions!

### DISCUSSION THREAD

**Discussion by stebo85** [neurodesk/neurodesk.github.io-hugo#226] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomEmotion** [comment]
Hello everyone. I am Tom Johnstone, Professor of Neuroimaging at Swinburne, and part of the AEDAPT team using Neurodesk at its core and planning to add electro- and magneto- analysis software to the platform.
